bal/p2p.py

```diff
@@ -55,21 +70,29 @@ def broadcast_latest(self):
         will request the entire chain if needed"""
         for peer_str in self.peer_sockets:
             peer_addr = self.get_peer_tuple(peer_str)
-            self.query(peer_addr, Message(MessageType.RESPONSE_BLOCKCHAIN, [self.blockchain().get_latest_block()], self.p2p_addr))
+            self.query(
+                peer_addr, Message(
+                    MessageType.RESPONSE_BLOCKCHAIN, [
+                        self.blockchain().get_latest_block()], self.p2p_addr))
```

Comment:
let's keep `Message` as a separate variable and then call it in the function to overcome this complicated indents.